src/select.tsx

```diff
@@ -408,56 +403,58 @@ const Select: React.FC<SelectProps> = ({
                     }}
                     disableScroll
                 >
```

Comment:
review this ignoring whitespace:
https://github.com/Telefonica/mistica-web/pull/802/files?diff=unified&w=1